hexrdgui/masking/mask_manager_dialog.py

```diff
@@ -408,10 +409,17 @@ def show_all_boundaries(self):
         self.update_presentation_selector()
         MaskManager().masks_changed()
 
-    def set_boundary_color(self):
-        dialog = QColorDialog(QColor(MaskManager().boundary_color), self.ui)
+    def edit_style(self):
+        dialog = MaskBorderStylePicker(
+            MaskManager().boundary_color,
+            MaskManager().boundary_style,
+            MaskManager().boundary_width
+        )
         if dialog.exec():
-            MaskManager().boundary_color = dialog.selectedColor().name()
+            color, style, width = dialog.result()
+            MaskManager().boundary_color = color
+            MaskManager().boundary_style = style
+            MaskManager().boundary_width = width
```

Comment:
So actually, since you now only set these if the dialog is accepted (the return of `dialog.exec()` is `True`), I think you don't even need to do the internal connection to the `reject` function since the values won't be modified if it is rejected. But it's not really an important change.